examples/skr/maa-test/README.md

```diff
@@ -0,0 +1,103 @@
+    - `confcom` Azure CLI extension (run `az extension add -n confcom`)
+- `gettext` (for `envsubst`)
+- A container registry (Azure Container Registry, Docker Hub, etc.)
+- An AKS cluster set up with the Kata-based preview
+    - Follow instructions here to get started: https://learn.microsoft.com/en-us/azure/aks/deploy-confidential-containers-default-policy
+
+## Get an MAA Token Workflow Step By Step
+
+### 1. Build the MAA test container
+
+Review the contents of [Dockerfile.maa_test](../../../docker/skr/Dockerfile.maa_test) and update the attestation endpoint if you wish.
+
+Below are the MAA endpoints for the four regions in which Confidential Containers on AKS is currently available.
+
+- East US: sharedeus.eus.attest.azure.net
+- West US: sharedwus.wus.attest.azure.net
+- North Europe: sharedneu.neu.attest.azure.net
+- West Europe: sharedweu.weu.attest.azure.net
+
+Run Docker build on the files from the root directory, replacing <SKR_MAA_IMAGE> with the desired container name and version:
+
+```bash
+docker build -t <SKR_MAA_IMAGE> -f docker/skr/Dockerfile.maa_test .
+```
+
+Push the container images to your container registry.
+
+### 2. Update the YAML file
+
+If you don't have a registry yet, you can follow the instructions in the [ACI README](../aci/README.md) to set one up.
+
+Replace the image value <SKR_MAA_IMAGE> in the [SKR MAA Comms YAML](skr-maa-comms-test.yaml) with the full registry plus container name of your image, i.e. `registry-name.azurecr.io/skr-maa-comms:1.0`.
+
+If you updated the attestation endpoint in [Dockerfile.maa_test](../../../docker/skr/Dockerfile.maa_test), update the SkrSideCarArgs value to reflect that. Base64 decode the current value, update, and base64 encode with padding before replacing the value in the YAML file.
+
+Create an image pull secret (this example uses an Azure Container Registry) so that kubectl can access the images and update the YAML with the secret:
+
+```bash
+export ACR_SECRET=<secret-name>
+kubectl create secret docker-registry $ACR_SECRET \
+    --namespace <namespace> \
+    --docker-server=<REGISTRY_NAME>.azurecr.io \
+    --docker-username=<appId> \
+    --docker-password=<password>
+
+envsubst < skr-maa-comms-test.yaml > skr-maa-comms-test.yaml
+```
+
+### 3. Generate a Security Policy
+
+The YAML file can be used directly to generate a security policy. The following command generates a security policy and automatically injects it into the YAML file. 
+
+```shell
+az confcom katapolicygen -y skr-maa-comms.yaml
+```
+
+In order to shell into or use kubectl logs after deploying, you have to update the security policy manually. Base64 decode the value in the template and find the following values:
+
+```text
+default ExecProcessRequest := false
+default ReadStreamRequest := false
+```
+
+Set these values to true and base64 encode with padding the updated security policy. Replace the previous value in the YAML file with the updated policy.
```

Comment:
`/bin/sh` or `/bin/bash` needs to be added to the list of allowed commands as well

Reply:
I don't think I've had to do that.